04-single-hop-6lowpan-icmp/example_test_guide.md

```diff
@@ -1,35 +1,34 @@
 _The following is a possible way to run the tests, this is just an example_
 ## Goal: Check single hop 6lowpan icmp
 
-_note: Since the ping6 function is now async some interval times may need to
-be increased to compensate_
-
 ### Pre-setup Steps
+_One can use IoT-LAB if for everything but the remote-revb boards_
 1. Get 2 of the following boards: `samr21-xpro`, `iotlab-m3`
 2. Get the following boards:`remote-revb`, `arduino-zero` with `xbee` module
-2. Go into the `cd ${RIOTBASE}/tests/gnrc_udp` directory
-2. Setup a terminal for each board _(tmux is your friend)_
-3. Find the serial numbers and ports with `ls /dev/serial/by-id/`
-4. For the samr21 terminals use the following to setup the env
+3. Go into the `cd ${RIOTBASE}/tests/gnrc_udp` directory
```

Comment:
Maybe it's just me, but "go into the" and the `cd` (which is litarally the command that translates to "go into a directory") seems weird to me.

Reply:
(weird as in redundant)